assets/src/scss/bootstrap-build.scss

```diff
@@ -21,8 +21,8 @@ Text Domain: planet4-master-theme
 @import "vendors/bootstrap";
 @import "~bootstrap/scss/functions";
 @import "~bootstrap/scss/variables";
-// The maps's file is required from bootstrap 5.2 and higher
-// @import "~bootstrap/scss/maps";
+@import "~bootstrap/scss/variables-dark";
```

Comment:
What is this import used for?

Reply:
It's used by `bootstrap/scss/maps`. Even if we haven't implemented the dark mode feature yet, the bootstrap's `$enable-dark-mode` is set to `true` by default within the [bootstrap/scss/_variables.scss](https://github.com/twbs/bootstrap/blob/c536836aa89e4d6a3f9003d3f7252fcb1a5be9bf/scss/_variables.scss#L387). So, it's better to include and avoid potential issues.